api/src/lists/daos/lists.dao.ts

```diff
@@ -124,53 +123,6 @@ export class ListsDao {
 				movie: true,
 			},
 		});
```

Comment:
We should probably remove this since we're getting movies from the `Movie` service.